docs/faqs/persist-predictions.mdx

```diff
@@ -5,6 +5,22 @@ sidebarTitle: Persisting Predictions
 
 MindsDB provides a range of options for persisting predictions and forecasts. Let's explore all possibilities to save the prediction results.
 
+<Note>
+**Reasons to Save Predictions**
+
+Every time you want to get predictions, you need to query the model, usually joined with an input data table, like this:
```

Comment:
Maybe we can explain that querying the model by default returns the result set which is not persistent. For future use, it is better to persist the result-set instead of querying the model again with the same data. What do you think?

Reply:
Good point. I'll add it up.